src/MicrosoftTeams.ts

```diff
@@ -741,6 +745,26 @@ function handleBeforeUnload(): void {
   }
 }
 
+/**
+ * Registers a handler for when the user reconfigurated tab
+ * @param handler The handler to invoke when the user click on Settings.
+ */
+export function registerOnSettingsChangeHandler(
```

Comment:
I think this name is slightly confusing since in my mind it implies that some settings have changed and this handler is the way that the tab can get notified. I wonder if a better name would be `registerChangeSettingsHandler` or `registerSettingsButtonClickedHandler` or something like that.

Reply:
I like `registerChangeSettingsHandler` better